package.json

```diff
@@ -34,14 +34,14 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "branch-diff": "^2.1.3",
+    "branch-diff": "^2.1.4",
     "chalk": "^5.3.0",
     "changelog-maker": "^3.2.4",
     "cheerio": "^1.0.0-rc.12",
     "clipboardy": "^3.0.0",
     "core-validate-commit": "^4.0.0",
     "enquirer": "^2.4.1",
-    "execa": "^7.2.0",
+    "execa": "^8.0.1",
```

Comment:
We should probably consider removing execa in favor of `run.js`

Reply:
Oh nevermind, I thought you meant a library called `run.js` but you actually meant the `run.js` file in the repository 😅.

Reply:
Yes, that's what I meant. Sorry I missed your comment.